lib/Doctrine/ORM/Mapping/ManyToMany.php

```diff
@@ -61,14 +62,22 @@ final class ManyToMany implements Annotation
      * @param array<string> $cascade
      */
     public function __construct(
-        string $targetEntity,
+        ?string $targetEntity = null,
```

Comment:
Why adding nullable when it gets deprecated?

Reply:
See the commit message :)

> There used to be no constructor in this class, adding one with mandatory
arguments was technically a BC-break.